clients/index.js

```diff
@@ -483,6 +484,13 @@ function updatePrunePeersPeriod() {
     self.serviceProxy.setPrunePeersPeriod(period);
 };
 
+ApplicationClients.prototype.updateConnectPeersPeriod =
+function updateConnectPeersPeriod() {
+    var self = this;
+    var period = self.remoteConfig.get('peerConnecter.period', 0);
```

Comment:
we shouldnt default to off.

If we cant talk to flipr we should default to like 100ms or something.